kubernetes/terraform/variables.tf

```
@@ -0,0 +1,46 @@
+variable "model_name" {
+  description = "Name of the juju model to deploy to"
+  type        = string
+}
+
+variable "app_name" {
+  description = "Name of the juju application"
+  type        = string
+  default     = "mysql-router"
```

```suggestion
  default     = "mysql-router-k8s"
```
to match default with `juju deploy`

---

Done in commit https://github.com/canonical/mysql-router-operators/pull/40/commits/80b0664ba54edeb96766f72863ffdf6c0d31dc67.